From 8bced38a941a181f1468fa39541e872e51b6022f Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Thu, 13 Jan 2022 08:14:13 +0000
Subject: [PATCH] lib: fix ax25.h include for musl

ax25.h isn't guaranteed to be avilable in netax25/*;
it's dependent on our choice of libc (it's not available
on musl at least) [0].

Let's use the version from linux-headers.

[0] https://sourceware.org/glibc/wiki/Synchronizing_Headers
Bug: https://bugs.gentoo.org/831102

Signed-off-by: Sam James <sam@gentoo.org>
Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
---
 lib/ax25_ntop.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/ax25_ntop.c b/lib/ax25_ntop.c
index cfd0e04b06f9..3a72a43e9b77 100644
--- a/lib/ax25_ntop.c
+++ b/lib/ax25_ntop.c
@@ -2,7 +2,7 @@
 
 #include <errno.h>
 #include <sys/socket.h>
-#include <netax25/ax25.h>
+#include <linux/ax25.h>
 
 #include "utils.h"
 
-- 
2.35.1

